cobbler/utils.py

```diff
@@ -216,9 +217,50 @@ def is_systemd() -> bool:
 
     This method currently checks if the path ``/usr/lib/systemd/systemd`` exists.
     """
-    if os.path.exists("/usr/lib/systemd/systemd"):
-        return True
-    return False
+    return os.path.exists("/usr/lib/systemd/systemd")
+
+
+def is_supervisord() -> bool:
+    """
+    Return whether or not this system uses systemd.
+
+    This method currently checks if the path ``/usr/bin/supervisord`` exists.
+    """
+    with ServerProxy('http://localhost:9001/RPC2') as server:
+
+      try:
+          server.supervisor.getState()
+      except OSError as e:
```

Comment:
Okay Codacy complains correctly that this variable is not used. So let's fix that:
```suggestion
      except OSError:
```